src/main/osd/osd.h

```diff
@@ -189,6 +189,14 @@ typedef enum {
     OSD_GPS_LAP_TIME_CURRENT,
     OSD_GPS_LAP_TIME_PREVIOUS,
     OSD_GPS_LAP_TIME_BEST3,
+    OSD_CUSTOM_MSG1,
```

Comment:
Not sure here. Maybe zero-indexing will be easier to read?

Reply:
OK